### PR TITLE
TCD/JDJ/RULE19-32

### PR DESCRIPTION
**Pull request by JacksonJ-KC**

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_32.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                                                        "id": "PTHP Terminal 1",
+                                                        "is_supply_ducted": false,
+                                                        "type": "CONSTANT_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "PTHP 1"
+                                                    }
+                                                ],
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "lighting_space_type": "COMPUTER_ROOM"
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_32.json:47]: Could you add the ```floor_area``` key here?

**Review by yunjoonjung-PNNL — APPROVED** [review]

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_32.json (hunk 1 above)
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_32.json:47]: Could you add the ```floor_area``` key here?

**Review by yunjoonjung-PNNL — CHANGES_REQUESTED** [review]

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_32.json (hunk 1 above)
Comment (JacksonJ-KC) [reply rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_32.json]: Added

**Review by yunjoonjung-PNNL — APPROVED** [review]
Looks great to me! Thank you.

**Review by yunjoonjung-PNNL — APPROVED** [review]
Looks great to me! Thank you.

**Pull request merged**